test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1

```diff
@@ -83,9 +85,9 @@ ScriptStackTrace : at <ScriptBlock>, <No file>: line 8
 Exception        :
     Type    : GetErrorPrimitiveArray
     Values  :
-        1
-        2
-        0.5
+              1
```

Comment:
This seems wrong it being indented even more than what it was before.

Reply:
Here's how it looks:

![image](https://github.com/jborean93/PowerShell/assets/11859881/f575ad40-1735-43ec-b4fa-094a95292518)

Reply:
Values are intended to be indented to align past the colon

Reply:
Fair enough.